api-reference/connect/desktop/web.mdx

```diff
@@ -1,241 +1,406 @@
 
+<Note>
+    Follow the quickstart tutorial to set up a project and find the required keys: [Create a new
+    project](/guides/wallet-as-a-service/waas/web-quickstart#configuring-particle-auth).
+</Note>
 
 ### Configuration
 
-With Particle Connect installed, you'll need to open your `index` file and wrap your `App` component with `ModalProvider`, imported from `@particle-network/connectkit`. Within `ModalProvider`, you'll have a variety of configurations to set:
-
-- `projectId`, `clientKey`, and `appId`, these are required values for configuring Particle Connect and can be retrieved from the [Particle dashboard](https://dashboard.particle.network/)
-
-In addition to the configurations within `options`, you can set:
-
-- `theme`, the theme of the connection UI, either `light`, `dark`, or `auto` (dependent on system settings).
-- `language`, the language used by the connection UI (optional.)
-- `walletSort`, the labels used for the different wallet categories (optional.)
-
-Below is a code snippet showcasing a structured configuration of `ModalProvider`, leveraging the parameters listed above.
-
-```typescript Particle Connect (Auth Core)
-import React from 'react'
-import ReactDOM from 'react-dom/client'
-import { Ethereum, EthereumGoerli } from '@particle-network/chains';
-import { ModalProvider } from '@particle-network/connectkit';
-import '@particle-network/connectkit/dist/index.css';
-import { evmWallets, solanaWallets } from '@particle-network/connectors';
-
-ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
-  <React.StrictMode>
-    <ModalProvider
-        options={{
-            projectId: process.env.REACT_APP_PROJECT_ID as string,
-            clientKey: process.env.REACT_APP_CLIENT_KEY as string,
-            appId: process.env.REACT_APP_APP_ID as string,
-            chains: [Ethereum, EthereumGoerli],
-            connectors: [
-                ...evmWallets({ projectId: process.env.REACT_APP_WALLETCONNECT_ID, showQrModal: true }),
-                ...solanaWallets(),
-            ],
-            erc4337: { //optional: account abstraction wallet UI config (displaying the smart account rather than EOA)
-              name: "SIMPLE",
-              version: "1.0.0"
-            },
-            wallet: { //optional: particle wallet config
-                customStyle: {
-                    supportChains: [Ethereum, EthereumGoerli],
-                },
-            },
-        }}
-    >
-        <App />
-    </ModalProvider>
-  </React.StrictMode>
-)
+With Particle Connect installed, you'll need to open your `index` file and wrap your `App` component with `ConnectKitProvider`, imported from `@particle-network/connectkit`. Within `ConnectKitProvider`, you'll have a variety of configurations to set:
```

Comment:
Here, we mention using the `ConnectKitProvider` in the `index` of the app, but then we show something slightly different with the code snippet. Might be a bit confusing.